app/js/index.js

```diff
@@ -26,6 +26,12 @@ $(document).ready(function() {
 
 
 function init() {
+  // don't initialize until user has uploaded a .csv file
+  if(document.getElementById('input_csv').files[0] == null){
+    console.log("Please upload a file");
```

Comment:
Should change to something like
"Cartograms 4 All: Waiting for user inputted file"